src/Data/Repositories/ChannelRepository.cs

```diff
@@ -62,12 +62,12 @@ IMapper mapper
 
             return await applicationDbContext.Channels.Include(x => x.ChannelOperationRequests).FirstOrDefaultAsync(x => x.Id == id);
         }
-        
+
         public async Task<Channel?> GetByChanId(ulong chanId)
         {
             await using var applicationDbContext = await _dbContextFactory.CreateDbContextAsync();
 
-            return await applicationDbContext.Channels.Include(x => x.ChannelOperationRequests).FirstOrDefaultAsync(x => x.ChanId == chanId);
+            return await applicationDbContext.Channels.Include(x => x.ChannelOperationRequests).FirstOrDefaultAsync(x => x.ChanId.Equals(chanId));
```

Comment:
The == operator is ok in this case, what's the goal of using equals?

Reply:
I was just discarding possible errors